benchmarks/advection_in_annulus/run_all_models

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Instructions for how to use this scipt are provided in the README.
```

Comment:
Currently there is no README. Add a short description here that mentions that this script runs all the model variations.